pkgs/development/interpreters/spidermonkey/common.nix

```diff
@@ -0,0 +1,184 @@
+{ version, hash }:
+
+{ callPackage
+, lib
+, stdenv
+, fetchurl
+, fetchpatch
+
+# build time
+, buildPackages
+, cargo
+, m4
+, perl
+, pkg-config
+, python3
+, python39
+, rustc
+, which
+, zip
+, autoconf213
+, yasm
```

Comment:
I noticed with conda's port that they didn't seem to need yasm so it might be safe to remove at least in 91+,

Reply:
@lostnet it is used only at 78 ;)